app/build.gradle

```diff
@@ -21,6 +22,7 @@ android {
         buildConfigField "String", "PARSE_APP_ID", apikeyProperties['PARSE_APP_ID']
         buildConfigField "String", "PARSE_CLIENT_KEY", apikeyProperties['PARSE_CLIENT_KEY']
         buildConfigField "String", "PARSE_SERVER_URL", apikeyProperties['PARSE_SERVER_URL']
+        buildConfigField "String", "GOOGLE_CLOUD_API_KEY", apikeyProperties['GOOGLE_CLOUD_API_KEY']
```

Comment:
is this needed if it is read from local.properties?  maybe a commend here stating where the key comes from instead.